### DISCUSSION TITLE
The space for lists on desktop is too big

### DISCUSSION THREAD

**Discussion by lauthieb** [community/community#8630] [Other Feature Feedback, Questions, & Ideas]
Hello! First of all, big thanks to Github for this awesome feature!!!
I've a small suggestion, it could be nice to reduce the size of the list on desktop because in my example, it's pretty big. Maybe we can move it on the right hand-side? I don't really know :-)

![Capture d’écran 2021-12-10 à 00 55 30](https://user-images.githubusercontent.com/9600228/145494363-57121f49-0e09-4ecd-8fcb-51d7c0090b7a.png)

Thanks!
Laurent

**Comment by efrecon** [comment]
I would suggest a tabular interface, if possible. One tab for "all stars", one tab for the list of lists, and perhaps the ability to "pin" a list or a few so they get their own tabs.
